README.md

```diff
@@ -1,5 +1,11 @@
 # Überauth Microsoft
 
+[![Module Version](https://img.shields.io/hexpm/v/ueberauth_microsoft.svg)](https://hex.pm/packages/ueberauth_microsoft)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/ueberauth_microsoft/)
+[![Total Download](https://img.shields.io/hexpm/dt/ueberauth_microsoft.svg)](https://hex.pm/packages/ueberauth_microsoft)
```

Comment:
Thanks for the PR. Can we drop the downloads badge please as this seems redundant since it directs to the same place as the version badge.

Reply:
@swelham Done. Please check.

Reply:
Thanks 👍